tools/ci/tc/decision.py

```diff
@@ -182,7 +190,7 @@ def build_full_command(event, task):
 
     return ["/bin/bash",
             "--login",
-            "-c",
+            "-xc",
```

Comment:
Deliberate change or leftover from debugging?

Reply:
Deliberate; it means we log the actual command that we're running.